src/app/actions.ts

```diff
@@ -34,12 +23,42 @@ const fuseOptions = {
   includeScore: true,
   keys: ["name"],
 };
-const fuse = new Fuse<FlattenedRegion>(flattenedRegionCodes, fuseOptions);
+const regionsFuseIndex = new Fuse<FlattenedRegion>(
+  flattenedRegionCodes,
+  fuseOptions,
+);
```

Comment:
Thanks for adding this! 🙏  something I'd meant to do after we discussed it and then forgor. Makes sense in this PR.

Reply:
Yep! tbh it probably wasn't necessary to do before now but since we've got two separate fuse indexes floating around now I think it makes sense to differentiate them more clearly.